### PR TITLE
Add target NBD_INFINITYAIO255

### PR DESCRIPTION
**Pull request by mywinchester**
firmware already verify and been tested in hardware.

### CODE REVIEW

File: configs/NBD_INFINITYAIO255/config.h [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+#define LED_STRIP_PIN        PA9
+#define UART2_TX_PIN         PA2
+#define UART3_TX_PIN         PB10
+#define UART7_TX_PIN         PE8
+#define UART8_TX_PIN         PE1
+#define UART1_RX_PIN         PB7
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PB11
+#define UART5_RX_PIN         PD2
+#define UART8_RX_PIN         PE0
```
Comment (haslinghuis) [inline configs/NBD_INFINITYAIO255/config.h:51]: Any wire diagram available?

We can assign functions to serial ports if needed when connectors are available.

Comment (haslinghuis) [reply configs/NBD_INFINITYAIO255/config.h:51]: TX: 2, 3, 7, 8
RX: 1, 2, 3, 5, 8

Needs clarification for missing TX1 and TX5 and RX7

Comment (LYNHQQ) [reply configs/NBD_INFINITYAIO255/config.h:51]: We will complete the missing UART definition later.

Comment (mywinchester) [reply configs/NBD_INFINITYAIO255/config.h:51]: > TX: 2, 3, 7, 8 RX: 1, 2, 3, 5, 8
> 
> Needs clarification for missing TX1 and TX5 and RX7

We using UART7 as analog video interface,  so it just using one pin.
And UART5/UART1 as ESC telemetry/SBUS, just need RX.

**Comment by sugaarK** [comment]
https://betaflight.com/docs/development/manufacturer/config-target-guidance
please read this @mywinchester

**Comment by mywinchester** [comment]
> https://betaflight.com/docs/development/manufacturer/config-target-guidance please read this @mywinchester

I had read this document, we will follow policy in next step.

**Comment by sugaarK** [comment]
opps sorry wrong button

**Comment by mywinchester** [comment]
> @mywinchester did you get NBD to email me or the target team? there is a new review process.

Not yet, but I will send ASAP

**Comment by mywinchester** [comment]
I already send an email to cloudtargets@betaflight.com, can you receive it? My email address is: mak@newbeedrone.com

**Comment by sugaarK** [comment]
@mywinchester  talking to Lexi from NBD you don't need this now as far as I know. ?

**Comment by nerdCopter** [comment]
@sugaarK , Discord had a couple people ask where the target was.  So the hardware is in the wild -- no clue how many.

**Comment by mywinchester** [comment]
> > @sugaarK , Discord had a couple people ask where the target was. So the hardware is in the wild -- no clue how many.
> 
> They have their own fork. They use a special osd that they don’t want to burden our project with

In fact this board osd is max7456 compatible, and this target `NBD_INFINITYAIO255` is already available in unified target repo, but somehow it can shows 4.5 version in configurator, that make people confuse. This pull request just wanna fix that problem. 
And as I know the PR open in one month ago, but [guide](https://betaflight.com/docs/development/manufacturer/config-target-guidance) said it should start at july. If this PR really bothers your team, please remove the target form configurator, and I will close PR.

**Comment by sugaarK** [comment]
There is no problem with it but every one has to pay the fee NBD can afford it and given no one else has a problem contributing to the project it shouldn’t be an issue.

**Comment by mywinchester** [comment]
@sugaarK  config file had been check and ready for release.

**Review by sugaarK — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**